components/slider/CHANGELOG.md

```diff
@@ -200,6 +200,20 @@ See [Conventional Commits](https://conventionalcommits.org) for commit guideline
 
 - **slider:**focus state matches spec, supports forced-colors ([#2217](https://github.com/adobe/spectrum-css/issues/2217))([7b9c31b](https://github.com/adobe/spectrum-css/commit/7b9c31b))
 
+### Migration Guide
+
+#### Indicating focus
+
+Focus must be bubbled up to the parent so descendants siblings can be styled.
+
+Thus, implementations should add the following class to the `.spectrum-Slider` parent class in the following situations:
+
+- `.is-disabled` - when the slider is disabled
+
+Implementations should also bubble the following class to the `.spectrum-Slider-controls` parent class in the following situations:
+
+- `.is-focused` - when the handle input is focused with the mouse or keyboard
```

Comment:
I did leave it out purposely from the docs page. Here's the Slack thread: https://adobedesign.slack.com/archives/G019JTYMT6H/p1721660043594259

The way we left it, it didn't sound like design know anything about with a "three handles" variant, so that bit was added to the changelog now, but I didn't put it on the docs page.